File: java64/nivel5/exer48.java
import javax.swing.JOptionPane;

public class exer48 {
    public static void main(String[] args) {
        int qtd = 20;
        String[] modeloNome = new String[qtd];
        int[] modeloIdade = new int[qtd];

        for (int i = 0; i < qtd; i++) {
            modeloNome[i] = JOptionPane.showInputDialog("Nome da " + (i + 1) +"° modelo");
            modeloIdade[i] = Integer.parseInt(JOptionPane.showInputDialog("Idade da " + (i + 1) +"° modelo"));
        }

        for (int x = 0; x < qtd; x++) {
            if (modeloIdade[x] >= 18 && modeloIdade[x] <= 20) {
                JOptionPane.showMessageDialog(null, "Modelo apta: " + modeloNome[x]);
            }
        }
    }
}

File: pyhton64/nivel5/exer48.py
modelo_nome = []
modelo_idade = []

for i in range(1, 20+1):
    modelo_nome.append(input('Nome:  '))
    modelo_idade.append((int(input('Idade: '))))

for z in range(len(modelo_idade)):
    if 18 <= modelo_idade[z] <= 20:
        print(f'Modelo apta: {modelo_nome[z]}')
